UI/RetroAchievementScreens.cpp

```diff
@@ -33,29 +47,102 @@ void RetroAchievementsListScreen::CreateTabs() {
 			unlockedAchievements.push_back(achievement);
 		}
 		return true;
-	});
+		});
 
 	achievements->Add(new ItemHeader(ac->T("Unlocked achievements")));
-	for (auto achievement : unlockedAchievements) {
-		achievements->Add(new AchievementView(achievement));
+	for (auto &achievement : unlockedAchievements) {
+		achievements->Add(new AchievementView(std::move(achievement)));
 	}
 	achievements->Add(new ItemHeader(ac->T("Locked achievements")));
-	for (auto achievement : lockedAchievements) {
-		achievements->Add(new AchievementView(achievement));
+	for (auto &achievement : lockedAchievements) {
+		achievements->Add(new AchievementView(std::move(achievement)));
+	}
+}
+
+void RetroAchievementsListScreen::CreateLeaderboardsTab(UI::ViewGroup *viewGroup) {
+	auto di = GetI18NCategory(I18NCat::DIALOG);
+	auto ac = GetI18NCategory(I18NCat::ACHIEVEMENTS);
+
+	using namespace UI;
+
+	viewGroup->Add(new GameAchievementSummaryView(Achievements::GetGameID()));
+
+	viewGroup->Add(new ItemHeader(ac->T("Leaderboards")));
+
+	std::vector<Achievements::Leaderboard> leaderboards;
+
+	Achievements::EnumerateLeaderboards([&](const Achievements::Leaderboard &leaderboard) {
+		leaderboards.push_back(leaderboard);
+		return true;
+		});
```

Comment:
nit: This happens a few times in this file, something I noticed VS 2022 is doing that I don't like.  If you also don't like it: I found that disabling "Indent braces of lambdas used as parameters" stops it from happening (although I might actually want that, just certainly not when the code and braces are indented the same which is the common-case result.)

-[Unknown]

Reply:
Oh, didn't realize there was an option for that! I will enable it.